sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -617,3 +617,47 @@ UPDATE `user_settings` SET `setting_value`='style_light.css' WHERE `setting_labe
  OR `setting_value`='style_purple.css'
  OR `setting_value`='style_radiant.css'
  OR `setting_value`='style_sky_blue.css');
+
+#IfNotColumnType facility country_code varchar(30)
+ALTER TABLE `facility` CHANGE `country_code` `country_code` varchar(30) NOT NULL default '';
```

Comment:
(right after brought into codebase noted the following :) )
It doesn't look like this change is addressed in database.sql ?
-brady

Reply:
I'll push a fix for it.  Thanks!